content/docs/forms.md

```diff
@@ -254,31 +254,31 @@ class Reservation extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/wgedvV?editors=0010)
+[**इसे CodePen पर आज़माएँ**](https://codepen.io/gaearon/pen/wgedvV?editors=0010)
 
-Note how we used the ES6 [computed property name](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names) syntax to update the state key corresponding to the given input name:
+ध्यान दीजिये की हमने दिए गए इनपुट नाम के लिए state key को अपडेट करने के लिए ES6 [कंप्यूटेड प्रॉपर्टी नाम](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names) सिंटेक्स का उपयोग किया है।
```

Comment:
` सिंटेक्स का उपयोग किया है।` => ` का सिंटेक्स का उपयोग किया है।`

Reply:
Can I write के सिंटेक्स का उपयोग किया है in place of का सिंटेक्स का उपयोग किया है